utils/get_intent_description.py

```diff
@@ -33,11 +36,13 @@ def add_output_arg(parser):
 
 
 def get_remote_workspace(args):
+    authenticator = IAMAuthenticator(args.password)
     conv = AssistantV1(
-        WA_API_VERSION,
-        username=args.user,
-        password=args.password
+        version=WA_API_VERSION,
+        authenticator=authenticator
     )
+    conv.set_service_url(BASE_URL)
```

Comment:
If this utility did not support a configurable URL, please create an Issue about that - this will not work when the Assistant is NOT located in US-South

Reply:
Added -l as a required argument for the Watson Assistant Url